app/views/components/phaseBanner.scala.html

```diff
@@ -1,30 +0,0 @@
-@*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
-@this(govukPhaseBanner: GovukPhaseBanner)
-
-@(phase: String, signOut: Boolean = true)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig)
-
-@feedbackBanner = {
-    @messages("feedback.before")
-    <a class='govuk-link' href='@if(signOut) @{appConfig.feedbackUrl} else @{appConfig.feedbackUnauthenticatedUrl}'>@messages("feedback.link")</a>
```

Comment:
feedbackUnauthenticatedUrl and feedbackUrl were removed but are still present in [here](https://github.com/hmrc/charities-registration-frontend/blob/DDCE-4394/app/config/FrontendAppConfig.scala#L43-L47). Please remove, thanks. Also, [gtmContainer](https://github.com/hmrc/charities-registration-frontend/blob/DDCE-4394/app/config/FrontendAppConfig.scala#L41) can be removed as it is unused. These [tests](https://github.com/hmrc/charities-registration-frontend/blob/DDCE-4394/test/config/FrontendAppConfigSpec.scala#L51-L61) will need removing.